<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>CASPy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/logo.png</normaloff>../assets/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_manager">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCopy">
    <property name="title">
     <string>Copy</string>
    </property>
    <addaction name="actionCopy_Exact_Answer"/>
    <addaction name="actionCopy_Approximate_Answer"/>
   </widget>
   <widget class="QMenu" name="menuTab">
    <property name="title">
     <string>Tab</string>
    </property>
    <addaction name="actionNext_Tab"/>
    <addaction name="actionPrevious_Tab"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionUnicode"/>
    <addaction name="actionLinewrap"/>
    <addaction name="separator"/>
    <addaction name="actionScientific_Notation"/>
    <addaction name="actionAccuracy"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuCopy"/>
   <addaction name="menuTab"/>
  </widget>
  <action name="actionCopy_Exact_Answer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Copy Exact Answer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPrevious_Tab">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Previous Tab</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionUnicode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Unicode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionNext_Tab">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Next Tab</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionLinewrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linewrap</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionScientific_Notation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scientific Notation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAccuracy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Accuracy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionClear_Shell">
   <property name="text">
    <string>Clear Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionCopy_Approximate_Answer">
   <property name="text">
    <string>Copy Approximate Answer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
